### DISCUSSION TITLE
Syntax highliting for svelte files

### DISCUSSION THREAD

**Discussion by Mehdi-YC** [NvChad/NvChad#1626] [Q&A] [answered]
Hello , please can someone tell me how can i enable syntax highliting for svelte , i saw that LSP does not have the highliting , thank you

**Comment by siduck** [comment, marked as answer]
@Mehdi-YC install svelt tsparser, `:TSInstall svelte`

**Comment by Mehdi-YC** [comment]
![image](https://user-images.githubusercontent.com/44881216/194845420-43865323-17e7-4c87-8066-003469db3b12.png)

this response is almost acceptable lol , the problem now is that it highlights the template part , but not the css & script ones , is this the limot if this highliter or a problem ?
![image](https://user-images.githubusercontent.com/44881216/194845466-e691fdab-9137-46f1-9bbd-dc36eff0cf51.png)

![image](https://user-images.githubusercontent.com/44881216/194845494-736aa794-9b1b-4f62-8091-2fd5fe000710.png)

**Reply by siduck** [reply]
install css tsparser

**Reply by Mehdi-YC** [reply]
yes this is what i was missing , thank you 
for every one also wondering 
:tsinstall css
:tsinstall javascript
